scripts/validate-tests

```diff
@@ -7,7 +7,7 @@ source ${SCRIPT_PATH}/utils/git
 
 header "Validating tests..."
 
-! egrep --quiet -R "fdescribe\\(|fit\(" ./src && \
-! egrep --quiet -R "context\.only\(|it\.only\(" ./tests
+! egrep -R "fdescribe\\(|fit\(" ./src || exit 1 && \
+! egrep -R "context\.only\(|it\.only\(" ./tests || exit 1
```

Comment:
@orlandohohmeier Why was the --quiet needed here?

Reply:
I'm also pretty sure that with `egrep` we don't need to escape the `(` like we do with `grep`.

Reply:
@bstavroulakis I have no idea why we decided to suppress the output and I think it's a good idea to not run the command in quite mode — though that's not the reason for the script failure.

@MatApple `egrep` is the same as `grep -E` and the search-pattern is regular expression thus we need to escape the parentheses. 

Side Note: `egrep` is actually only there for historic reasons and we should switch to `grep -E`